EIPS/eip-5875.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 5875
+title: Opcode for transaction index
+description: Add an opcode to retrieve transaction index in block
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/eip-5875-opcode-for-transaction-number-in-a-block/11612
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-11-04
+---
+
+## Abstract
+
+This EIP adds an instruction, `OPCODE_TBD (transactionNUM)` to retrieve the Transaction Index inside of a block.
```

Comment:
```suggestion
This EIP adds an instruction `TXINDEX` to retrieve the Transaction Index inside of a block.
```

Not sure the need for the TBD mnemonic here. Please also choose a specific opcode number. It can be changed if this EIP is considered for an upgarde.